src/app/home/home.style.scss

```diff
@@ -0,0 +1,66 @@
+.home {
+
+    &-nav {
+        height: 70px;
+
+        &-menu {
+            float: right;
+            margin: 30px 80px 0 0;
+
+            &-list {
+                font-size: 0.8em;
+                font-weight: 500;
+                &__title {
+
+                }
+
+                &__item {
+                    height: 10px;
+                    line-height: 10px;
+                    display: inline-block;
+                    padding: 0 5px;
+                    border-right: 1px solid #d1d1d1;
+                    color: #d1d1d1;
+                    cursor: pointer;
+
+                    &:last-child {
+                        border-right: none;
+                    }
+                }
+            }
+        }
+
+        &__create {
```

Comment:
I think this is not the element, this is block ```button``` block
you can leave this element only for context-styling like color, placing, floating ....